resources/js/components/Search/Autocomplete.vue

```diff
@@ -1,121 +1,94 @@
-            // Initialize with empty data to preserve additionals order
-            self.results = Object.fromEntries(Object.keys(self.additionals).map((indexName) => [indexName, []]))
-            self.resultsCount = 0
+            return client
+        },
 
+        searchkit: function () {
             let url = new URL(config.es_url)
-            let auth = `Basic ${btoa(`${url.username}:${url.password}`)}`
-            let baseUrl = url.origin
 
-            Object.entries(self.additionals).forEach(([name, data]) => {
-                let stores = data['stores'] ?? null
-                if (stores && !stores.includes(window.config.store_code)) {
-                    return
-                }
-
-                let fields = data['fields'] ?? data
-                let size = data['size'] ?? self.size ?? undefined
-                let sort = data['sort'] ?? undefined
-
-                let multimatch = self.multiMatchTypes.map((type) => ({
-                    multi_match: {
-                        query: query,
-                        type: type,
-                        fields: fields,
-                        fuzziness: type.includes('phrase') ? undefined : 'AUTO',
-                    },
-                }))
-
-                let esQuery = {
-                    size: size,
-                    sort: sort,
-                    query: {
-                        bool: {
-                            should: multimatch,
-                            minimum_should_match: 1,
-                        },
-                    },
-                    highlight: {
-                        pre_tags: ['<mark>'],
-                        post_tags: ['</mark>'],
-                        fields: Object.fromEntries(fields.map((field) => [field.split('^')[0], {}])),
-                        require_field_match: false,
+            let searchkit = new Searchkit({
+                connection: {
+                    host: url.origin,
+                    auth: {
+                        username: url.username,
+                        password: url.password,
                     },
-                }
-
-                // es_prefix has been removed, Scout is used
-                rapidezFetch(`${baseUrl}/${config.es_prefix}_${name}_${config.store}/_search`, {
-                    method: 'POST',
-                    headers: { 'Content-Type': 'application/json', Authorization: auth },
-                    body: JSON.stringify(esQuery),
-                }).then(async (response) => {
-                    const responseData = await response.json()
+                },
 
-                    self.results[name] = responseData?.hits ?? []
-                    self.results.count += self.results[name]?.hits?.length ?? 0
-                })
+                // TODO: Maybe just do: search_settings: config.searchkit
+                // so it's possible to add anything to the PHP config
+                // and that will appear here?
+                search_settings: {
```

Comment:
```suggestion
                // TODO: Should we split this as it could
                // diff from the settings on listings.
                search_settings: {
```